src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/settings/ProjectSettingsRemoteAjaxController.java

```diff
@@ -0,0 +1,136 @@
+package ca.corefacility.bioinformatics.irida.ria.web.projects.settings;
+
+import java.security.Principal;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.MessageSource;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+import ca.corefacility.bioinformatics.irida.model.RemoteAPI;
+import ca.corefacility.bioinformatics.irida.model.project.Project;
+import ca.corefacility.bioinformatics.irida.model.project.ProjectSyncFrequency;
+import ca.corefacility.bioinformatics.irida.model.remote.RemoteStatus;
+import ca.corefacility.bioinformatics.irida.model.user.User;
+import ca.corefacility.bioinformatics.irida.ria.web.ajax.dto.RemoteProjectSettings;
+import ca.corefacility.bioinformatics.irida.ria.web.ajax.dto.RemoteProjectSettingsUpdateRequest;
+import ca.corefacility.bioinformatics.irida.service.ProjectService;
+import ca.corefacility.bioinformatics.irida.service.remote.ProjectRemoteService;
+import ca.corefacility.bioinformatics.irida.service.user.UserService;
+
+import com.google.common.collect.ImmutableMap;
+
+/**
+ * Controller for managing settings for a remotely sync'd project
+ * asynchronously
+ */
+@RestController
+@RequestMapping("/ajax/remote-projects/{projectId}/settings")
+public class ProjectSettingsRemoteAjaxController {
+	private MessageSource messageSource;
```

Comment:
Where's the UI service?  You are making this controller so messy.

Reply:
Added in 3bd93720a5eb17ec1c86cc4c22cfa47f30d13312